<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageSize</class>
 <widget class="QDialog" name="DialogImageSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Image Size</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblLabelImageWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbUnitsWidth"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblLabelImageHeight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="cmbUnitsHeight"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtImageWidth"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtImageHeight"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabelMouseTracker" name="lblImage" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>&lt;h4&gt;Compartment geometry image&lt;/h4&gt;
&lt;p&gt;An image of the geometry of the model.&lt;/p&gt;</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblLabelPixelSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Size of a pixel:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="lblPixelSize">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelMouseTracker</class>
   <extends>QWidget</extends>
   <header>qlabelmousetracker.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtImageWidth</tabstop>
  <tabstop>cmbUnitsWidth</tabstop>
  <tabstop>txtImageHeight</tabstop>
  <tabstop>cmbUnitsHeight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
